lib/runner.js

```diff
@@ -99,7 +105,8 @@ Runner.prototype._addTestResult = function (result) {
 		error: result.reason,
 		type: test.metadata.type,
 		skip: test.metadata.skipped,
-		todo: test.metadata.todo
+		todo: test.metadata.todo,
+		exclusive: test.metadata.exclusive
```

Comment:
It's not necessary. That's me mixing up two wholly different reasons for the full test suite failing. Removing in a jiffy!